python_bindings/test/correctness/compile_to.py

```diff
@@ -43,10 +43,14 @@ def main():
         else:
             assert os.path.isfile(os.path.join(tmpdir, "f_all.o"))
 
-        p = os.path.join(tmpdir, "f.html")
+        p = os.path.join(tmpdir, "f.stmt.html")
```

Comment:
Should this be `.stmt.viz.html`

Reply:
No, this is (still) testing the older `stmt_html` output (which may be deprecated/removed, see #7507)